src/Resources/Resources/RoleAssignments/NewAzureRoleAssignmentCommand.cs

```diff
@@ -318,6 +318,13 @@ public override void ExecuteCmdlet()
                     Id = ObjectId,
                     ObjectType = ObjectType,
                 },
+                ResourceIdentifier = new ResourceIdentifier() {
+                    ParentResource = ParentResource,
+                    ResourceGroupName = ResourceGroupName,
+                    ResourceName = ResourceName,
+                    ResourceType = ResourceType,
+                    Subscription = DefaultProfile.DefaultContext.Subscription.Id,
```

Comment:
Some partners call this at the tenant level and this value becomes null.
Please add null coallesing operators to fall back to "" (it is not ised on this module so it should not affect our workflows)